ansible/roles/cartodb.resque/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+ - name: Ensure pid file dir exists
+   command: mkdir -p {{ cartodb_resque_pid_dir }}
```

Comment:
Snazzy. Will definitely make this change, it slid under the radar with all the other things I was changing/copypasting